src/lua/zencode_sd_jwt.lua

```diff
@@ -416,6 +416,21 @@ When("use signed selective disclosure '' only with disclosures ''", function(ssd
     ssd.disclosures = disclosure
 end)
 
+IfWhen("verify selective disclosure '' contains disclosures ''", function(ssd_name, lis)
```

Comment:
more aligned with other statements to use:

`is found` or `are found`

`verify disclosures '' are found in selective disclosure ''`

perhaps we can make `is` and `are` equivalent

Reply:
Used `are found`